docs/getting-started/install-crossplane.md

```diff
@@ -0,0 +1,83 @@
+---
+toc_hide: true
+---
+
+## Install Crossplane
+
+Crossplane installs into an existing Kubernetes cluster. 
+
+{{< hint type="tip" >}}
+If you don't have a Kubernetes cluster create one locally with [minikube](https://minikube.sigs.k8s.io/docs/start/) or [kind](https://kind.sigs.k8s.io/).
```

Comment:
This looks great! My concern--undoubtedly biased from just having wasted way too much time trying to get minikube working--is that this removes the previous `kind` happy path instructions for macOS/Homebrew users that were the default. I think new users would be better served--and Crossplane would receive fewer "I can't get this working" reports (and the ones it does receive would have a variable eliminated)--by only suggesting `kind` to users who don't yet have Kubernetes set up.

Maybe something llke "If you don't already have a local Kubernetes cluster (eg minikube) or run into trouble we recommend using kind."

Reply:
I'm 100% okay with that. I'll let @hasheddan and @sftim chime in too before I make the change-

Any objection to only recommending Kind due to recent minikube issues? ([minikube issues #14505](https://github.com/kubernetes/minikube/issues/14505) and [minikube issues  #14523](https://github.com/kubernetes/minikube/issues/14523))

Reply:
I made a thing while procrastinating this afternoon. https://gist.github.com/fuzz/40d345bda5c6e52b4b0a1635520dee37

Reply:
> If you don't already have a local Kubernetes cluster (eg minikube) or run into trouble we recommend using kind."

Seems fair. The Kubernetes project tries to be extra neutral on recommending / not recommending cluster setup tools; projects like Crossplane might not need to be. Now that kind doesn't require Docker it's even more reasonable to recommend kind, IMO.

Reply:
+1 I’m fine with recommending kind as the happy path here